### DISCUSSION TITLE
Additions to the default layout

### DISCUSSION THREAD

**Discussion by Eloquencere** [zellij-org/zellij#3622] [General]
Hi, I really like the default layout of zellij, I want to make some improvements that will improve my experience, I want to:
* Move the session name to the mode-list line, only when in the session mode (to the right side)
* Centre the tabs on top.

I want almost everything else to remain the same, is there any way, I can do it with the default tools given by zellij, or do I have to fully re-create the layout via `zjstatus` and then implement my improvements? If so, how would I go about doing it?

**Comment by giyany** [comment]
You definitely want a plugin for that. I'm not sure if zjstatus can achieve everything, but can say that it's very popular. 
Once you need help with zjstatus you can ask directly there or join the zellij Discord or Matrix, where people are super helpful with specifics.

**Comment by Eloquencere** [comment]
Oh wow, that's a lot of effort. I doubt people having a configuration of the zellij default layout. Since most people install zjstatus _because_ they want to depart from the standard out-of-the-box layouts.

Maybe I'll try opening an issue on it since it'll be a bit easier for the devs to make that decision, they'll probably see some rationale behind it, since it would end up de-cluttering the UI and freeing up some space for more tabs to be visible in the same view.

**Reply by giyany** [reply]
The devs' approach is to encourage the community to write and use plugins rather than extending the core functionality of zellij.
Whatever you choose,  it's useful to add that `jzstatus` (and also `zj-status-bar`) is actively and thoughtfully maintained and worth trying out.
